lemur/certificates/views.py

```diff
@@ -694,6 +694,9 @@ def get(self, certificate_id):
         response = make_response(jsonify(key=cert.private_key), 200)
         response.headers["cache-control"] = "private, max-age=0, no-cache, no-store"
         response.headers["pragma"] = "no-cache"
+
+        log_service.audit_log("export_private_key", cert.name,
+                              "Private key exported for the certificate")
```

Comment:
shall we use a consistent language? like `Exported private key.`

Reply:
specially looking how the log will appear we can go with a common theme:
https://github.com/Netflix/lemur/blob/3783fbeaa1645bbee022827f4f53ffb12dd65a61/lemur/logs/service.py#L47